README.md

```diff
@@ -5,13 +5,14 @@ WSO2 Balana Implementation
 
 |  Branch | Build Status |
 | :------------ |:-------------
-| master      | [![Build Status](https://wso2.org/jenkins/job/wso2-balana/badge/icon)](https://wso2.org/jenkins/job/wso2-balana) |
-
+| master      | [![Build Status](https://wso2.org/jenkins/buildStatus/icon?job=forked-dependencies/wso2-balana)](https://wso2.org/jenkins/job/forked-dependencies/wso2-balana) |
```

Comment:
@madurangasiriwardena Your jenkins must have been unavailable yesterday when I looked because the image link didn't work in @mefarazath 's repository. Now it works again.